airflow/providers/google/cloud/operators/mysql_to_gcs.py

```diff
@@ -102,10 +102,12 @@ def field_to_bigquery(self, field):
     def convert_type(self, value, schema_type):
         """
         Takes a value from MySQLdb, and converts it to a value that's safe for
-        JSON/Google Cloud Storage/BigQuery. Dates are converted to UTC seconds.
-        Decimals are converted to floats. Binary type fields are encoded with base64,
-        as imported BYTES data must be base64-encoded according to Bigquery SQL
-        date type documentation: https://cloud.google.com/bigquery/data-types
+        JSON/Google Cloud Storage/BigQuery. Datetimes are converted to UTC seconds.
```

Comment:
Can you write these conversations as a bulleted list? This will allow us to find a specific case more easily.

Reply:
Done! Can I get another review please? 😃